modules/assim.sequential/inst/paleon_sda.R

```diff
@@ -1,26 +1,49 @@
 
 library(PEcAn.all)
 library(PEcAn.SIPNET)
+library(PEcAn.LINKAGES)
```

Comment:
I thought we were deprecating library calls? Is there any reason not to use namespace calls?

Reply:
While I don't disagree with @mccabete, it's worth noting that this is a script in the inst folder, not a function definition, so I'm not as worried about library calls here